lib/roby/interface/rest/server.rb

```diff
@@ -194,10 +204,10 @@ def server_alive?
                 #
                 # @raise InvalidServer if there is a server at the expected
                 #   host and port, but not a Roby REST server
-                def self.server_alive?(host, port)
+                def self.server_alive?(host, port, main_route)
```

Comment:
Set to '/api' by default here

Reply:
I stick to the host/port pattern, where the default is only on the constructor. Sure you want this argument to have a default here? @doudou

Reply:
Actually, you've got a point. The port could also have a default.

But more importantly, this breaks backward compatibility for no strong enough reason.